nuxeo-features/nuxeo-platform-comment/nuxeo-platform-comment/src/test/java/org/nuxeo/ecm/platform/comment/AbstractTestCommentManager.java

```diff
@@ -319,6 +356,186 @@ public void testCommentManagerType() {
         assertEquals(getType(), commentManager.getClass());
     }
 
+    @Test
+    public void shouldNotifyEventWhenCreateComment() {
+        DocumentModel domain = session.createDocumentModel("/", "domain", "Domain");
+        session.createDocument(domain);
+        DocumentModel doc = session.createDocumentModel(domain.getPathAsString(), "test", "File");
+        doc.addFacet(SubscriptionAdapter.NOTIFIABLE_FACET);
```

Comment:
Needed? Not present in the next test.

Reply:
good point it's needed see [here](https://github.com/nuxeo/nuxeo/blob/f7f5104b9d212b270b8a7ffb769d78e23d896fe7/nuxeo-features/nuxeo-platform-notification/nuxeo-platform-notification-core/src/main/java/org/nuxeo/ecm/platform/ec/notification/SubscriptionAdapter.java#L71) it's why i added for creation method, but i will check it before adding it